docs/add-merge-automation-to-your-repository.md

```diff
@@ -0,0 +1,38 @@
+# Adding merge automation to your repository
+
+In order to run tests and enable automated merging of pull requests for your repository on KubeVirt [prow] you need to have the following components configured:
```

Comment:
I think it would be good to distinguish here between actual Prow components (tide and branch-protector) and plugins (lgtm, approve, trigger), wdyt?

Reply:
I didn't want to go into too much detail, but you are right, let's be clear here.